packages/utils/src/phoneNumbers.ts

```diff
@@ -191,6 +191,9 @@ function handleSpecialCasesForDisplay(parsedNumber: PhoneNumber, countryCode?: n
         .format(parsedNumber, PhoneNumberFormat.INTERNATIONAL)
         .replace(/\+54(\s)?/, '')
 
+    case 231:
+      return phoneUtil.format(parsedNumber, PhoneNumberFormat.NATIONAL).slice(1)
```

Comment:
I'd feel more comfortable only slicing if the lead digit is 0. Worried that the auto 0 insertion won't happen for all Liberian numbers